apps/researcher/src/app/[locale]/(objects)/objects/[id]/metadata.tsx

```diff
@@ -6,29 +6,31 @@ import {ChatBubbleBottomCenterTextIcon} from '@heroicons/react/24/solid';
 import {SlideOutButton, SlideOut} from '@colonial-collections/ui';
 import {UserEnricherForm} from './user-enrichment-form';
 import {SignedIn} from '@clerk/nextjs';
-import {fetcher} from '@/lib/enricher-instances';
-import {getCommunityByAttributionId} from '@/lib/community/actions';
 import {getFormatter} from 'next-intl/server';
 import classNames from 'classnames';
 import {InformationCircleIcon} from '@heroicons/react/24/outline';
+import {AdditionalType} from '@colonial-collections/enricher/src/definitions';
 
-const useMetadata = create(() => ({
+const useMetadata = create<{
+  translationKey: string;
+  additionalType?: AdditionalType;
+}>(() => ({
   translationKey: '',
-  enrichmentIdentifier: '',
+  additionalType: undefined,
```

Comment:
Question: is `additionalType` an OK name for you? `additionalType` makes sense in the context of the `enricher`, but it might not be here (where the context is missing)?